rollup-operator/src/server/operator.js

```diff
@@ -179,36 +179,56 @@ const portAdmin = process.env.OPERATOR_PORT_ADMIN;
 
 appAdmin.post("/loadwallet", async (req, res) => {
     const walletObj = req.body.wallet;
-    const wallet = await ethers.Wallet.fromEncryptedJson(walletObj, req.body.pass);
-    opManager.loadWallet(wallet);
-    res.sendStatus(200);
+    try {
+        const wallet = await ethers.Wallet.fromEncryptedJson(walletObj, req.body.pass);
+        opManager.loadWallet(wallet);
+        res.sendStatus(200);
+    } catch (error) {
+        res.send(400).status(`Message error: ${error.message}`);
```

Comment:
Consideration: return a human readable error response (example: "error loading wallet"). And log the full error trace on the console. I get that this app is for admins, but it seems a bit extreme to send full errors to the frontend

Reply:
Agree! xD
I would review all the http responses together to improve messages, either errors or information. Since there are several endpoints I would do it in a separate branch after we define proper messages